src/styles/blocks/product-catalog.scss

```diff
@@ -0,0 +1,29 @@
+.product-catalog {
+  padding-block: 50px;
+  padding-inline: 40px;
+
+  display: grid;
+  grid-template-columns: repeat(1, minmax(200px, 200px));
```

Comment:
Consider using `1fr` instead of `minmax(200px, 200px)` for `grid-template-columns` as they are functionally equivalent in this context, but `1fr` is simpler and more intuitive.